frontend/catalyst/pennylane_extensions.py

```diff
@@ -193,8 +215,8 @@ def grad(f, *, method=None, h=None, argnum=None):
 
     .. warning::
 
-        Currently, higher-order differentiation is only supported by the
-        finite-difference method.
+        Currently, higher-order differentiation or differentiation of non-QNode functions
```

Comment:
What's a different between "non-QNode" and "arbitrary" functions in this context?

Reply:
They mean the same thing to me, although technically arbitrary would include QNode as well as non-QNode functions.

Reply:
yeah! I guessed so. Would you like to make this consistent in the description of other methods as well? I'd personally like "non-QNode" functions better than arbitrary :)